<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="6" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="3738" started-at="2018-11-18T11:59:28Z" finished-at="2018-11-18T11:59:32Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="3738" started-at="2018-11-18T11:59:28Z" finished-at="2018-11-18T11:59:32Z">
      <class name="com.lala.test.RunTest">
        <test-method status="PASS" signature="verifyPlaceOrder()[pri:1, instance:com.lala.test.RunTest@643b1d11]" name="verifyPlaceOrder" duration-ms="2812" started-at="2018-11-18T17:29:28Z" finished-at="2018-11-18T17:29:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyPlaceOrder -->
        <test-method status="PASS" signature="verifyFetchOrderWhenOrderExist()[pri:2, instance:com.lala.test.RunTest@643b1d11]" name="verifyFetchOrderWhenOrderExist" duration-ms="221" started-at="2018-11-18T17:29:31Z" finished-at="2018-11-18T17:29:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyFetchOrderWhenOrderExist -->
        <test-method status="FAIL" signature="verifyFetchOrderWhenOrderDoesNotExist()[pri:3, instance:com.lala.test.RunTest@643b1d11]" name="verifyFetchOrderWhenOrderDoesNotExist" duration-ms="30" started-at="2018-11-18T17:29:31Z" finished-at="2018-11-18T17:29:31Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Test is failed, status code not matched please check you GET request payload]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Test is failed, status code not matched please check you GET request payload
	at org.testng.Assert.fail(Assert.java:94)
	at com.lala.test.requests.FetchOrder.verifyStatusCode(FetchOrder.java:39)
	at com.lala.test.RunTest.verifyFetchOrderWhenOrderDoesNotExist(RunTest.java:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyFetchOrderWhenOrderDoesNotExist -->
        <test-method status="PASS" signature="verifyDriveToTakeOrder()[pri:4, instance:com.lala.test.RunTest@643b1d11]" name="verifyDriveToTakeOrder" duration-ms="202" started-at="2018-11-18T17:29:31Z" finished-at="2018-11-18T17:29:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyDriveToTakeOrder -->
        <test-method status="PASS" signature="verifyDriveToComplete()[pri:5, instance:com.lala.test.RunTest@643b1d11]" name="verifyDriveToComplete" duration-ms="221" started-at="2018-11-18T17:29:32Z" finished-at="2018-11-18T17:29:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyDriveToComplete -->
        <test-method status="PASS" signature="verifyCancelOrder()[pri:6, instance:com.lala.test.RunTest@643b1d11]" name="verifyCancelOrder" duration-ms="191" started-at="2018-11-18T17:29:32Z" finished-at="2018-11-18T17:29:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyCancelOrder -->
      </class> <!-- com.lala.test.RunTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
